src/features/components/molecules/instruments-table/instrumental-table.module.scss

```diff
@@ -0,0 +1,27 @@
+@use 'features/styles/theme/theme-mixins' as *;
+@import 'features/styles/theme/variables';
+
+.instrumental_wrapper {
+    border: 1px solid $color-grey-6;
+    flex-direction: column;
+
+    @include breakpoints(tablet) {
+        flex-direction: row;
+    }
+
+    .instrumental_left_item {
+        border-bottom: 1px solid $color-grey-6;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        @include breakpoints(tablet) {
+            width: 20%;
+            border-right: 1px solid $color-grey-6;
```

Comment:
here too

Reply:
i coudnt find a variable name for this 
src/features/styles/theme/buttons.scss
saw similar usage in this file too